Automattic/wp-calypso
# Update ETK phpunit to work with new wp-env improvements

#### Proposed Changes
In https://github.com/WordPress/gutenberg/pull/41780 and https://github.com/WordPress/gutenberg/pull/41852 @ObliviousHarmony made some improvements to wp-env we can take advantage of in Calypso:

1. wp-env now installs the WP test files for the WordPress version installed, removing the need for wp-phpunit.
2. the `phpunit` container was deprecated upstream. We can control the phpunit version ourselves with composer. Since this container was also broken on M1 machines, not using it will fix that issue.

This shouldn't be merged until the wp-env update is available on npm. 

#### Testing Instructions
Run `$ path/to/gutenberg/packages/env/bin/wp-env run tests-wordpress /var/www/html/wp-content/plugins/editing-toolkit-plugin/vendor/bin/phpunit -c /var/www/html/wp-content/plugins/editing-toolkit-plugin/phpunit.xml.dist --verbose` and it should pass.
